stdlib/public/core/BridgeObjectiveC.swift

```diff
@@ -85,6 +85,22 @@ public protocol _ObjectiveCBridgeable {
 
 #if _runtime(_ObjC)
 
+@available(macOS, introduced: 9999, deprecated)
+@available(iOS, introduced: 9999, deprecated)
+@available(watchOS, introduced: 9999, deprecated)
+@available(tvOS, introduced: 9999, deprecated)
```

Comment:
This is *also* not valid syntax. :-/ Fortunately the `*` deprecation should still apply.

Reply:
Ok well the compiler accepts this one so I guess I found a bug